src/content/app/genome-browser/components/zmenu/ZmenuController.tsx

```diff
@@ -71,12 +70,20 @@ const ZmenuController = (props: Props) => {
     <Zmenu
       key={id}
       zmenuId={id}
-      browserRef={props.browserRef}
+      containerRef={props.containerRef}
       payload={zmenus[id]}
     />
   ));
 
   return <>{zmenuElements}</>;
 };
 
+const isZmenuPayload = (
+  payload: HotspotPayload
+): payload is ZmenuCreatePayload => {
+  // NOTE: check for payload.variety.length should be unnecessary;
+  // but as of now, genome brwoser can't separate track's left corner hotspot from transcript zmenu hotspot
```

Comment:
`browser` is misspelt in the comment above

Reply:
Updated.

But another **NOTE TO MYSELF:** this comment should be unnecessary after Dan fixes the hotspots code (and the line of code below the comment should be updated accordingly).